crime-applications-adaptor/src/main/resources/schemas/crimeapplicationsadaptor/common/childWeighting.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "childWeighting.json",
+  "type": "object",
+  "description": "Assessment Child Weighting",
+  "properties": {
+    "noOfChildren": {
+      "type": "integer",
+      "description": "Number of children"
+    },
+    "lowerAgeRange": {
+      "type": "integer",
+      "description": "Lower age range",
+      "enum": [0, 2, 5, 8, 11, 13, 16]
+    },
+    "upperAgeRange": {
+      "type": "integer",
+      "description": "Upper age range",
+      "enum": [1, 4, 7, 10, 12, 15, 18]
+    }
```

Comment:
Same as above, I think these are all required. If the lower and upper age range aren't filled out then we can't map the noOfChildren